### PR TITLE
Support `LastLedgerSequence` in transactions

### PR DESCRIPTION
**Pull request by keefertaylor**
- Update `xpring-common-js`
- Wire a new RPC for retrieving the latest ledger sequence
- When sending, retrieve the last ledger sequence, pad by a constant amount, and add to `lastLedgerSequence` 
- Add tests

### CODE REVIEW

File: src/grpc-network-client.ts [hunk 1]
```diff
@@ -68,6 +70,23 @@ class GRPCNetworkClient implements Networking.NetworkClient {
...
   }
+
+  public async getLatestValidatedLedgerSequence(
+    getLatestValidatedLedgerSequenceRequest: GetLatestValidatedLedgerSequenceRequest
+  ): Promise<LedgerSequence> {
+    return new Promise((resolve, reject): void => {
+      this.grpcClient.getLatestValidatedLedgerSequence(
+        getLatestValidatedLedgerSequenceRequest,
+        (error, response): void => {
+          if (error != null || response == null) {
```
Comment (Stormtv) [inline src/grpc-network-client.ts]: `!= null` and `== null` checks if the left side value is `null or undefined` if you want to strictly check `null` then use `===`

File: src/xpring-client.ts [hunk 2]
```diff
@@ -146,68 +149,84 @@ class XpringClient {
...
-            );
-          }
-
-          const submitSignedTransactionRequest = new SubmitSignedTransactionRequest();
-          submitSignedTransactionRequest.setSignedTransaction(
-            signedTransaction
-          );
+          return this.getLastValidatedLedgerSequence().then(
+            async ledgerSequence => {
+              if (accountInfo.getSequence() == undefined) {
```
Comment (Stormtv) [inline src/xpring-client.ts]: `== undefined` checks if the getSequence() value is `null or undefined` if you want to strictly check `undefined` then use `===`

File: src/xpring-client.ts [hunk 3]
```diff
@@ -146,68 +149,84 @@ class XpringClient {
...
 
   /* eslint-enable no-dupe-class-members */
 
+  private async getLastValidatedLedgerSequence(): Promise<number> {
+    const getLatestValidatedLedgerSequenceRequest = new GetLatestValidatedLedgerSequenceRequest();
+    const ledgerSequence = await this.networkClient.getLatestValidatedLedgerSequence(
+      getLatestValidatedLedgerSequenceRequest
+    );
+    const index = ledgerSequence.getIndex();
+    return index;
```
Comment (Stormtv) [inline src/xpring-client.ts]: Is there a reason to not return on line 226?

File: src/xpring-client.ts (hunk 2 above)
Comment (keefertaylor) [reply src/xpring-client.ts]: Done

**Pull request merged**